qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/services/QuteCodeActions.java

```diff
@@ -72,6 +72,10 @@
 
 	private static final String EXCLUDED_VALIDATION_TITLE = "Exclude this file from validation.";
 
+	private static final String SET_IGNORE_SEVERITY_TITLE = "Change the `{0}` setting severity to ignore.";

```

Comment:
@fbricon  what do you think about this title?

Reply:
An idea of title should be

"Ignore `undefined object` diagnostic"

Reply:
I was thinking it might be a good idea to work have a constants file, since I'm going to be adding all the different validation settings.

Reply:
CodeAction message has been changed to "Ignore `qute.validation.undefinedObject.severity` diagnostic."

Reply:
"Ignore {0} problem"